presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -727,29 +727,45 @@ private static DynamicFiltersResult createDynamicFilters(
             return clausesBuilder.build();
         }
 
+        private static boolean isComparisonOperator(CallExpression call, JoinNode node)
+        {
+            if (call.getDisplayName().equals(EQUAL.getFunctionName().getObjectName()) ||
```

Comment:
Please use `OperatorType.isComparisonOperator`. DisplayName is just for display purpose. Logic should not be built based on it.

Reply:
sure. get it fixed